can-stache-bindings.js

```diff
@@ -642,7 +642,16 @@ var behaviors = {
 			};
 			canReflect.onValue(bindingContextObservable, updateListener);
 		} else {
-			canEventQueue.on.call(bindingContext, event, handler);
+			try {
+				canEventQueue.on.call(bindingContext, event, handler);
+			} catch (error) {
+				var msg = 'can-stache-bindings - Unable to bind "' + event + '"';
```

Comment:
Does this have the same problem you recently fixed in `can-observable-mixin`... is this going to catch errors not related to this binding issue and throw a confusing error?

Reply:
I didn't find a test example that can be caught by the `try ... catch` bloc, but I added a regex check for the right error message.